pkg/util/computeskus/computeskus.go

```diff
@@ -77,3 +77,16 @@ func FilterVMSizes(skus []mgmtcompute.ResourceSku, location string) map[string]*
 
 	return vmskus
 }
+
+// SupportedOSDisk returns the type of OSDisk for the given resource. Most VMs will use Premium disks but some SKUs only support Standard SSDs
+func SupportedOSDisk(vmSku *mgmtcompute.ResourceSku) string {
+	const (
```

Comment:
is there a reason why this block is directly in the function ?

Reply:
Just declaring them in the area they're used, if you have a suggestion on a better place to put them I'd be happy to hear it. From what I could find we don't have these defined as a type anywhere like we do VMSizes.

Reply:
So, I first asked that because I just found it a little curious to have a block like that, so it was more out of curiosity. But I just found that you write them again in the test file, while you would be able to just use them if they were declared outside of the function. I would be a little cleaner this way IMO. (talking about lines 265-275 in the test file)

Reply:
Your English is fine. Better than my French at the very least ;)

Files updated.